<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="testcases.TestSample" duration-ms="9683" started-at="2020-07-21T21:54:59Z" finished-at="2020-07-21T21:55:09Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="9683" started-at="2020-07-21T21:54:59Z" finished-at="2020-07-21T21:55:09Z">
      <class name="testcases.TestSample">
        <test-method status="PASS" signature="setupEnvironment()[pri:0, instance:testcases.TestSample@246ae04d]" name="setupEnvironment" is-config="true" duration-ms="2906" started-at="2020-07-21T21:54:59Z" finished-at="2020-07-21T21:55:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupEnvironment -->
        <test-method status="FAIL" signature="multiplyTest()[pri:0, instance:testcases.TestSample@246ae04d]" name="multiplyTest" duration-ms="3010" started-at="2020-07-21T21:55:02Z" finished-at="2020-07-21T21:55:05Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Display is 72] but found [Display is 0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Display is 72] but found [Display is 0]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at testcases.TestSample.multiplyTest(TestSample.java:49)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- multiplyTest -->
        <test-method status="PASS" signature="sumTest()[pri:0, instance:testcases.TestSample@246ae04d]" name="sumTest" duration-ms="2798" started-at="2020-07-21T21:55:05Z" finished-at="2020-07-21T21:55:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sumTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:testcases.TestSample@246ae04d]" name="tearDown" is-config="true" duration-ms="949" started-at="2020-07-21T21:55:08Z" finished-at="2020-07-21T21:55:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.TestSample -->
    </test> <!-- Command line test -->
  </suite> <!-- testcases.TestSample -->
</testng-results>
